about.html

```diff
@@ -13,51 +13,80 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+      <!--header with all Lambda logo and Navbar included-->
```

Comment:
There is a semantic element called header you can use specifically for what you are doing with the article.